.release/linux/preremove

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+case "$1" in
+    remove | 0)
```

Comment:
do we care about the `upgrade` case here too?

Reply:
In this case, no, because we don't want to stop/disable anything if we're doing an upgrade -- the postremove handles restarting things, so we just let the upgrade case fall through. Normal package upgrade behavior, barring specific reasons to do things a different way, is "leave things running, swap out for the new version, then conditionally restart".